acceptance/bundle/telemetry/deploy-variable-count/databricks.yml

```diff
@@ -0,0 +1,32 @@
+bundle:
+  name: chicken
+
+variables:
+  v1:
+    description: var 1
+    default: 1
+
+  v2:
+    description: var 2
+    default: 2
+
+  v3:
+    description: var 3
+    default: 3
+
+  lv1:
+    description: lookup var 1
+    lookup:
+      cluster: cluster_one
+
+  lv2:
+    description: lookup var 2
+    lookup:
+      cluster: cluster_two
+
+  c1:
+    description: complex 1
+    type: complex
```

Comment:
What if you omit "type: complex" line? It's still a complex variable (value is a map), so it should be tracked as such.

Reply:
The `complex` annotation has to be statically set. We don't do runtime detection of a complex variable.
https://github.com/databricks/cli/blob/6fe8acb4da738241439b8ee1c68796849eb16914/bundle/config/variable/variable.go#L82

Without the annotation complex variables will not work.

Reply:
> The complex annotation has to be statically set.

If you have default, you can determine if it's complex. Otherwise it would undercount complex variables but there is no indication of so in the data. Alternative is to call this "annotatedComplex" or something like that.

IsComplex() function is wrong here.